.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -19,7 +19,9 @@ SHELL [ "/bin/bash", "-c" ]
 
 ARG OWLBOT_CLI_COMMITTISH=ac84fa5c423a0069bbce3d2d869c9730c8fdf550
 ARG PROTOC_VERSION=25.3
+ARG GRPC_VERSION=1.62.2
```

Comment:
Can we configure this with renovate bot, and group it with other grpc update? e.g. There is an existing [PR](https://github.com/googleapis/sdk-platform-java/pull/3034) that updates gRPC to the latest.

Reply:
Done

https://github.com/googleapis/sdk-platform-java/blob/9795e23af1aeab77d6c9557000c311e21f972978/renovate.json#L55-L67

Reply:
Done